### PR TITLE
issue 163: Updated documentation of remaining methods for creation of table

### PR DESCRIPTION
**Pull request by CodTan**
Added examples for methods: from_records(), read_table(), from_df(), from_array()

[ ] Wrote test for feature
[ ] Added changes to CHANGELOG.md
[ ] Bumped version number (delete if unneeded)

**Changes proposed:**
<description of the PR>

### CODE REVIEW

File: datascience/tables.py [hunk 1]
```diff
@@ -115,11 +129,52 @@ def from_columns_dict(cls, columns):
 
     @classmethod
     def read_table(cls, filepath_or_buffer, *args, **vargs):
-        """Read a table from a file or web address.
+        """Read a table from a file or web address."""
```
Comment (davidwagner) [inline datascience/tables.py:132]: What's the purpose/effect of changing this line?

Comment (CodTan) [reply datascience/tables.py:132]: There's no effect. Misinterpreted a previous PR, my bad!

Comment (CodTan) [reply datascience/tables.py]: I've fixed the extra quotes in the last commit.

File: datascience/tables.py [hunk 2]
```diff
@@ -175,11 +237,29 @@ def from_array(cls, arr):
...
+        ...       ('A',1), ('B',2)], 
+        ...       dtype=[('Name', 'U10'), ('Number', 'i4')]
+        ...       )
+                         
+        >>> print(arr)
+        [('A', 1) ('B', 2)]
+        
+        >>> t = Table().from_array(arr)
+        
+        >>> print(t)
```
Comment (adnanhemani) [inline datascience/tables.py:258]: Same as above comment - please remove the call to `print` function :)

**Review by adnanhemani — CHANGES_REQUESTED** [review]

**Comment by adnanhemani** [comment]
Thanks for this contribution! Please see my comments for additional required changes :)

**Comment by CodTan** [comment]
I have introduced the changes as requested :)

**Review by adnanhemani — APPROVED** [review]

**Comment by adnanhemani** [comment]
Thanks for the changes!

**Pull request merged**